src/schema/v2/partnerArtworksMutation.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  GraphQLBoolean,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import {
+  GravityMutationErrorType,
+  formatGravityError,
+} from "lib/gravityErrorHandler"
+import { GraphQLObjectType } from "graphql"
+import { InternalIDFields } from "./object_identification"
+import { GraphQLUnionType } from "graphql"
+interface Input {
+  id: string
+  artsyShippingDomestic: boolean | null
+  artsyShippingInternational: boolean | null
+  location: string | null
+}
+
+const PartnerArtworksType = new GraphQLObjectType<any, ResolverContext>({
```

Comment:
This type might be better named as `UpdatePartnerArtworksMutationSuccessDetails` or something? (caveat is that's verbose and I'm not a great namer 😆 )

Reply:
You are totally right! I've updated this as the name is verbose but explains exactly what it is. Also fits with the existing types.